final_assignment/alex-mitan/conf.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+# *-* coding: UTF-8 *-*
+
+from __future__ import print_function
+
+import ast
+import os
+import sys
+import time
+import urllib
+
+# the raw content of the config file
+
+build_log = open("build.log", "w")
+
+
+def func_config(cfg_path):
+    # Phase -1 - Set variables
+    build_log.write("Working platform: " + sys.platform + '\n')
+
+    if sys.platform.lower().startswith("win"):
+        work_os = "windows"
+    elif sys.platform.lower().startswith("linux"):
+        work_os = "linux"
+
+    build_log.write("Platform interpreted as: " + work_os + '\n' + '\n')
+    # Phase 0 - Parse dict from file
+
+    # cut out the actual portion of the string for the config dictionary
+    cfg_raw = open(cfg_path, "r").read()
+
+    dict_start = cfg_raw.index("{")
+    dict_end = len(cfg_raw) - 1
+    while cfg_raw[dict_end] != "}":
+        dict_end -= 1
+
+    cfg_raw = cfg_raw[dict_start:dict_end + 1]
+    cfg_dict = dict(eval(cfg_raw))
+
+    if cfg_dict:
+        build_log.write("Config dictionary successfully parsed")
```

Comment:
faci write in fisier dar nu adaugi si un endline, tot ce scrii o sa fie pe aceeasi linie.